ci/prod/build/backend.Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.24-bullseye
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends zip
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)